Analyse.cs

```diff
@@ -3,34 +3,149 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using CMP1903M_Assessment_1.Debugging;
 
 namespace CMP1903M_Assessment_1
 {
+    /// <summary>
+    /// Stores and analyses the user input
+    /// </summary>
     public class Analyse
     {
-        //Handles the analysis of text
+        private string _text = string.Empty;
+        public string Text
+        {
+            get { return _text; }
+            private set { _text = value; }
+        }

```

Comment:
This could be simplified using [autoproperties](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties).

```suggestion
        public string Text { get; private set; }
```